versioned_docs/version-8.7/self-managed/identity/application-user-group-role-management/applications.md

```diff
@@ -0,0 +1,82 @@
+---
+id: applications
+title: "Applications"
+sidebar_label: "Applications"
+---
+
+In Identity, an Application represents an entity that can request Identity to authenticate a user or a service.
+
+For Camunda 8 to work, a set of Applications are preconfigured. These preconfigured applications are:
+
+- Connectors
+- Identity
+- Operate
+- Optimize
+- Tasklist
+- Web Modeler
+- Zeebe
+
+As a user of Identity, you can add own Applications.
+
+## Types of applications
+
+There are three types of applications in Identity:
```

Comment:
Maybe here as well we can mention that this follows the OAuth protocol? We have a "Note" box below, but it's not very obvious.

Reply:
Fair. Added